README.md

```diff
@@ -63,14 +63,14 @@ require_once(__DIR__ . '/vendor/autoload.php');
 // Configure OAuth2 access token for authorization: oauth2
 SquareConnect\Configuration::getDefaultConfiguration()->setAccessToken('YOUR_ACCESS_TOKEN');
 
-$api_instance = new SquareConnect\Api\CatalogApi();
-$body = new \SquareConnect\Model\BatchDeleteCatalogObjectsRequest(); // \SquareConnect\Model\BatchDeleteCatalogObjectsRequest | An object containing the fields to POST for the request.  See the corresponding object definition for field details.
+$api_instance = new SquareConnect\Api\ApplePayApi();
+$body = new \SquareConnect\Model\RegisterDomainRequest(); // \SquareConnect\Model\RegisterDomainRequest | An object containing the fields to POST for the request.  See the corresponding object definition for field details.
```

Comment:
Should it be made clear here that you're changing the docs so that the ONLY "getting started" example is *specific* to using ApplePay, and that the README no longer contains an example *without* using ApplePay?

Reply:
Thanks for the feedback. I will improve the readme in a different PR. This code is autogenerated, and I need to update the generator template to make this better.